data_discovery_ai/model/keywordModel.py

```diff
@@ -45,31 +40,38 @@
 logger.setLevel(logging.INFO)
 
 
-def get_class_weights(Y_train):
+"""
+    Calculate label weights by the frequency of a label appears in all records
+    Input:
+        Y_train: numpy.ndarray. The train set of Y
+    Output:
+        label_weight_dic: Dict[int, float]. The label weights, keys are the indexs of labels and values are the weights.
+"""
```

Comment:
these should be inside the method, not above

Reply:
Moved inside the methods, changes applied to all functions.